DESCRIPTION

```diff
@@ -36,13 +36,13 @@ Imports:
     magrittr,
     glue (>= 1.4.2),
     rlang (>= 0.4.10),
-    SASxport (>= 1.8.0),
     SnowballC (>= 0.7.0),
```

Comment:
Do we sitll need this if we moved xport_varnaems to dev for now?

Reply:
We can add that in in the varnames pr